vicamo/actions-library
# chore(main): release 1.0.0

:robot: I have created a release *beep* *boop*
---


## 1.0.0 (2024-04-09)


### 🚀 Features

* add debian-releases action ([9db02fd](https://github.com/vicamo/actions-library/commit/9db02fd9f62a68e2d2afa63638d4605a324ff125))


### 🧰 Testing

* enable dependabot ([18d944f](https://github.com/vicamo/actions-library/commit/18d944fe9bebb5775e93eefaf0ebbabe1d362fe2))
* enable release-please ([7cadb2b](https://github.com/vicamo/actions-library/commit/7cadb2bd496ca9ac3848452cdeb0ffb43092669e))
* initial CI support with prettier ([71545fc](https://github.com/vicamo/actions-library/commit/71545fceb2dfcbfc173826d4e6765aebef4b9557))
* release-please: set type to simple ([7e7f299](https://github.com/vicamo/actions-library/commit/7e7f2993847b63b03212b793db15cc63656b5420))

---
This PR was generated with [Release Please](https://github.com/googleapis/release-please). See [documentation](https://github.com/googleapis/release-please#release-please).
